### DISCUSSION TITLE
Setting to enable / disable beeper on menu selections

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#445] [Ideas]
Right now when you select options on the menu the beeper triggers, would be nice silent for night-time operation as it's quite loud.

**Comment by Jyers** [comment]
That would be easy enough, I'll look into it.

**Reply by VashBaldeus** [reply]
> 
> 
> That would be easy enough, I'll look into it.

Just for the Menu beeps, not for Errors or issue that arise, those are alarms that require it to beep.
